internal/sftpclient/sftpclient.go

```diff
@@ -4,13 +4,13 @@ import (
 	"fmt"
 	"io"
 	"io/ioutil"
-	"log"
 	"os"
 	"path"
 	"path/filepath"
 	"time"
 
 	"github.com/fusor/cpma/pkg/config"
+	"github.com/fusor/cpma/pkg/log"
```

Comment:
@alexander-demichev I didn't notice you did this for config too... IIRC, pkg/* is the code which is Ok to use in external projects. This is not the case. We have internal/* for this.